test/exitstrategies/test_main.py

```diff
@@ -0,0 +1,22 @@
+"""
+Test for dummy lockdown vector
+"""
+
+from help_project.src.exitstrategies.interface import ExitStrategies
```

Comment:
instead of test_main can we raname to test_interface as we should plan on adding a test/commit hook that cheks appropriate test files have been added automatically. At that point having `test_filename` to check against will be helpful!

Reply:
This is done